Mathlib/LinearAlgebra/Matrix/Ideal.lean

```diff
@@ -5,20 +5,24 @@ Authors: Jujian Zhang, Wojciech Nawrocki
 -/
 import Mathlib.Data.Matrix.Basis
 import Mathlib.RingTheory.TwoSidedIdeal.Operations
+import Mathlib.RingTheory.JacobsonIdeal
```

Comment:
This seems to be tagged with `large-import`. I initially split out the Jacobson results into a separate file, but decided it was too small to deserve a standalone existence.

Reply:
The impirt increase is *huge* though. Can you find a better way? maybe with a pereliminary split to an existing file?

Reply:
What do you mean by 'preliminary split to an existing file'?

Reply:
Oh I hadn't noticed it xas a leaf file. Then >i'd say it's fine